### ISSUE TITLE
[regression in 0.22.0] Blank screen on start

### ISSUE THREAD

**Issue by yurivict** [extrawurst/gitui#1437]
**Describe the bug**
When gitui is started it causes the whole screen to go blank.
Ctrl-C doesn't do anything.
killing the process causes terminal settings to be altered. This terminal then requires reset.

```
$ env | grep TERM
COLORTERM=truecolor
TERM=xterm-256color
```

Xfce-4
FreeBSD 13.1

**Comment by extrawurst** [comment]
Please run with logs enabled my share the log for diagnostics

**Comment by yurivict** [comment]
Diagnostics log only has these lines:
```
08:43:01 [TRACE] (6) mio::poll: [/disk-samsung/freebsd-ports/devel/gitui/work/gitui-0.22.0/cargo-crates/mio-0.8.4/src/poll.rs:521] registering event source with poller: token=Token(0), interests=READABLE
08:43:01 [TRACE] (6) mio::poll: [/disk-samsung/freebsd-ports/devel/gitui/work/gitui-0.22.0/cargo-crates/mio-0.8.4/src/poll.rs:521] registering event source with poller: token=Token(1), interests=READABLE
08:43:01 [TRACE] (1) mio::poll: [/disk-samsung/freebsd-ports/devel/gitui/work/gitui-0.22.0/cargo-crates/mio-0.8.4/src/poll.rs:521] registering event source with poller: token=Token(0), interests=READABLE
```

**Comment by extrawurst** [comment]
can you build from source as a debug build?

**Comment by extrawurst** [comment]
Can you please give the current master a try by building from source via `make debug`? the generated log will help me figure out what's going on

**Comment by yurivict** [comment]
The latest revision now works without this problem.

**Comment by yurivict** [comment]
If/once the problem is resolved - please make another release.

**Comment by extrawurst** [comment]
closed via bbcadcb
